plugins/drill-coverage-plugin/src/adminPartMain/kotlin/AgentState.kt

```diff
@@ -68,15 +76,16 @@ class AgentState(
             .onEach { analyzer.analyzeClass(it.second, it.first) }
             .toMap()
         val bundleCoverage = coverageBuilder.getBundle("")
-        val prevData = agentData.prevData
         val currentMethods = classBytes.mapValues { (className, bytes) ->
             BcelClassParser(bytes, className).parseToJavaMethods()
         }
-        val prevMethods = prevData?.javaMethods?: mapOf()
-        val diff = MethodsComparator(bundleCoverage).compareClasses(prevMethods, currentMethods)
+        var prevData = agentData.prevData
+        val (uncheckedDiff, uncheckedChanged) = calculateDiff(prevData, bundleCoverage, currentMethods)
```

Comment:
So awwful.
put this "**val (uncheckedDiff, uncheckedChanged) = calculateDiff(prevData, bundleCoverage, currentMethods)**" calculation into " **val (diff, changed)**" assignment.
And ofc we can create a structure with 3 fields or else. and fixed this **var** on 82 line